app/models/search_builder.rb

```diff
@@ -31,4 +31,10 @@ def search_choice(question, choice)
       }
     }
   end
+
+  def advanced_search
+    client = ElasticCustomClient.new
+    query_string = %Q({"query":{"match":{"body.analyzed": "hack"}}}")
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.